### PR TITLE
Universal links

### PR DESCRIPTION
**Pull request by uerceg**
* Updating README for universal link support instructions.

### CODE REVIEW

**Comment by uerceg** [comment]
@adjust/copy

File: README.md [hunk 1]
```diff
@@ -297,17 +297,48 @@ In the Project Navigator open the source file your Application Delegate. Find
...
-- (BOOL)  application:(UIApplication *)application openURL:(NSURL *)url
-    sourceApplication:(NSString *)sourceApplication annotation:(id)annotation
-{
+- (BOOL)application:(UIApplication *)application openURL:(NSURL *)url
+  sourceApplication:(NSString *)sourceApplication annotation:(id)annotation {
     [Adjust appWillOpenUrl:url];
     
     // Your code goes here
-    Bool canHandle = [self someLogic:url];
+    BOOL canHandle = [self someLogic:url];
```
Comment (MrPleaze2015) [inline README.md]: //if-i-could-stand-itself-would-u-mind?

File: README.md [hunk 2]
```diff
@@ -440,12 +471,14 @@ You can read more about special partners and these integrations in our
...
 
 ## License
 
 The adjust SDK is licensed under the MIT License.
 
-Copyright (c) 2012-2015 adjust GmbH,
+Copyright (c) 2012-2016 adjust GmbH,
 http://www.adjust.com
 
 Permission is hereby granted, free of charge, to any person obtaining a copy of
```
Comment (MrPleaze2015) [inline README.md]: FREEDOM'FREEDOM Oh yes! FREEDOM'FREEDOM Oh yes, yes, yes!
FREEDOM'FREEDOM wOw YES!!!
Thankful to be Grateful,,,
We are1FOREVER 
We have jUan(wOn)2geder 
WISDOM'PASSION'LEGACY

File: README.md [hunk 3]
```diff
@@ -440,12 +471,14 @@ You can read more about special partners and these integrations in our
...
 [currency-conversion]: https://docs.adjust.com/en/event-tracking/#tracking-purchases-in-different-currencies
+[universal-links-guide]: https://docs.adjust.com/en/universal-links/
+[universal-links]: https://developer.apple.com/library/ios/documentation/General/Conceptual/AppSearch/UniversalLinks.html
 
 ## License
 
 The adjust SDK is licensed under the MIT License.
 
-Copyright (c) 2012-2015 adjust GmbH,
+Copyright (c) 2012-2016 adjust GmbH,
```
Comment (MrPleaze2015) [inline README.md]: http://git.io/BaH8pQ

**Pull request merged**